src/domain/users/user-organizations.repository.interface.ts

```diff
@@ -43,12 +43,14 @@ export interface IUsersOrganizationsRepository {
     users: Array<{
       address: `0x${string}`;
       role: UserOrganization['role'];
+      name: UserOrganization['name'];
```

Comment:
Nit: as before, I'd also document why this takes a name parameter. i.e.: a name is "proposed" by the admin, but might be changed by the invited user during the acceptance process (I if understood correctly)